src/Dotnet.Script.Tests/EnvironmentTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using System.Text.RegularExpressions;
+
+
+namespace Dotnet.Script.Tests
+{
+    public class EnvironmentTests
+    {
+        [Fact]
+        public void ShouldPrintVersionNumber()
+        {
+            var result = ScriptTestRunner.Default.Execute("--version");
+            Assert.Equal(0, result.exitCode);
+            Assert.Matches(@"\d*.\d*.\d*", result.output);
```

Comment:
This is an incorrect test. It says match 3 _Unicode digit_ separated by _anything_ and which appears _anywhere_ in the output. What's more, since the Unicode digits are _optional_ (`*`), it will also match 3 of anything! The right test would be:

```c#
Assert.Matches(@"^[0-9]+(\.[0-9]+){2}$", result.output);
```

However, this test is still somewhat inaccurate. If you run `dotnet script --version`, the output is:

```

0.23.0
```

The trouble here is that there is a leading blank line! The test shouldn't be passing but looks like it does because `ProcessHelper.RunAndCaptureOutput` trims the output:

https://github.com/filipw/dotnet-script/blob/9bced12676415fd5eb1c69b95cf3765c71190358/src/Dotnet.Script.Tests/ProcessHelper.cs#L40

Ideally trimming should be decision of the caller.